src/scripts/main.js

```diff
@@ -1,3 +1,22 @@
 'use strict';
 
-// write code here
+const header = document.querySelector('tr');
+const rows = document.querySelector('tbody');
```

Comment:
Use 'querySelectorAll' instead of 'querySelector' to select all 'tbody' elements.